docs/server-admin.md

```diff
@@ -0,0 +1,37 @@
+# The administration panel
+
+Iodide has an administration view under `/admin`, based on the [Django
+admin](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/) interface.
+
+To log into this interface, a user must either have the `is_staff` or `is_superuser` permission set. Most operations (including the one currently
+described here) require `is_superuser`.
+
+An existing super user can grant old users super-user permissions via
+the panel. Or, if no super user is currently configured, you can perform this operation manually via a SQL session (`./manage.py dbshell` from the console):
+
+```sql
+update base_user set is_superuser='t' where username=MY_USER_NAME;
+```
+
+## Authentication tokens
+
+It may, in some cases, by desirable to create special users which can
+create notebooks on behalf of others (e.g. bots). For this task, we
+have configured the iodide server to allow for the notebook creation API to be accessed using django-rest-framework's [token authentication])(https://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication)
+
+To take advantage of this:
+
+1. Go to the server's admin panel (as a user with root permissions, see above) and create a new user with the "can create on behalf of
```

Comment:
Even with `is_staff` and `is_superuser` permissions, I don't see a way to create new users.

![Screenshot showing what the admin interface looks like to me](https://user-images.githubusercontent.com/933396/54644048-c0063100-4a6e-11e9-87f2-a5cefa5e67da.png)

Reply:
I created this account / log into it with GitHub, if that matters.